pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -527,7 +526,7 @@ func (os *OpenStack) LoadBalancer() (cloudprovider.LoadBalancer, bool) {
 		glog.V(3).Infof("Using LBaaS extension %v", lbVersion)
 	}
 
-	glog.V(1).Info("Claiming to support LoadBalancer")
```

Comment:
See https://github.com/kubernetes/community/blob/master/contributors/devel/logging.md for the guide on severities.  I feel these "Claiming to support ..." statements indicate "information about config" since they are (or were?) printed only once at startup as confirmation that the plugin had successfully found and enabled significant chunks of functionality.  I think these should stay as `V(1)`, but I'm willing to be convinced otherwise.

Reply:
yeah, it is called in `init` function of servicecontroller, changed it back.